src/ape_ethereum/multicall/exceptions.py

```diff
@@ -0,0 +1,25 @@
+from ape.exceptions import ApeException
+
+
+class MulticallException(ApeException):
```

Comment:
I wonder if the base exception would make sense at the root exceptions levels so Starknet could also use it.

Reply:
Added to #1217

Reply:
Also add to future refactoring work into EcosystemAPI